frontend/src/components/PageListing.tsx

```diff
@@ -17,35 +17,35 @@ const PageCard: React.SFC<PageCardProps> = (props) => {
 
 type PageListingProps = {
   zine: Zine
-  children: ReactElement | undefined
 }
 
-const PageListing: React.FC<PageListingProps> = (props) => {
+const PageListing: React.FC<PropsWithChildren<PageListingProps>> = props => {
   const [pages, setPages] = useState<Page[]>([])
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(true)
 
   useEffect(() => {
-    fetchPages(props.zine, setLoading).then((json)=>setPages(json))
+    fetchPages(props.zine, setLoading).then(json => setPages(json))
   }, [props.zine.id])
 
   return (
-    <div className="flex grow overflow-y-auto justify-start flex-col items-center gap-6 w-screen">
-      <div className="bg-yellow-200">
-        {props.zine.name}
-      </div>
-      {
-        loading ? <p>Loading</p>:
-          <div className="flex flex-start flex-wrap justify-center gap-4 overflow-y-auto max-h-fit w-9/12">
-        {
-          pages.map((page) => {
-            return(<div className="shrink-0" key={"zine_"+ props.zine.id +"_page_" + page.id}><PageCard page={page}></PageCard></div>)
-          })
-        }
+    <div className='flex grow overflow-y-auto justify-start flex-col items-center gap-6 w-screen'>
+      <div className='bg-yellow-200'>{props.zine.name}</div>
+      {loading ? (
+        <p>Loading</p>
+      ) : (
+        <div className='flex flex-start flex-wrap justify-center gap-4 overflow-y-auto max-h-fit w-9/12'>
+          {pages.map(page => {
+            return (
+              <div className='shrink-0' key={`zine_${props.zine.id}_page_${page.index}`}>
```

Comment:
Prettier didn't do the auto backtick formatting; put it back, if you prefer the other way, and it won't get overridden